### DISCUSSION TITLE
Energy spectrum for electron detector

### DISCUSSION THREAD

**Discussion by SudeNurKeles** [ozel/DIY_particle_detector#57] [Q&A]
Hello, ı am currently working on the electron detector, and I want to observe the energies of beta particles by analyzing the signals. Is it possible to do so by using the codes you have provided for the electron detector, or do I need to work with different methods or codes? Thank your for your attention.

**Comment by ozel** [comment]
yes, all analysis scripts that I used for doing the scientific paper are in the repo, have you read the paragraphs around figure 7 about analysing betas (and few gammas) from KCL/potassium salt? 
https://www.mdpi.com/1424-8220/19/19/4264

please keep in mind that betas and gammas or x-rays are only *partially* absorbed beyond ~10 keV of energy (see figure 1 in the paper), that means they'll loose also energy in the surrounding passive (non-sensitive) material of the diodes and will likely also exit the diodes and travel further with remaining energy
